content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -171,9 +171,7 @@ using the (deprecated) v1alpha2 API instead.
 
 This section outlines the necessary steps to use containerd as CRI runtime.
 
```

Comment:
I think we might need to add a third-party warning here per Tim's suggestion on the issue. @sftim could you please confirm if that was the intended outcome or did I mistake it for something else?

Reply:
@sftim Any suggestions?

Reply:
There's `{{% thirdparty-content %}}` on line 186